### ISSUE TITLE
NuxtApp is overwritten. Other plugins lose types.

### ISSUE THREAD

**Issue by Jassmin586** [victorgarciaesgi/nuxt-typed-router#47]
In generated file `{project_name}/generated/typed-router.d.ts` custom declaration appeared. This code overwrites NuxtApp interface. So others plugins lose their typings. Can I stop generate this declare?

![code](https://user-images.githubusercontent.com/19594765/197950776-a279e754-fc63-42ba-9650-6275d323708d.png)

With this declare:
![image](https://user-images.githubusercontent.com/19594765/197951434-b5243048-8950-495c-841f-885abc17af53.png)

Without this declare:
![image](https://user-images.githubusercontent.com/19594765/197951596-f9eb59c3-94c2-4c58-9fc2-03001fc7ee9c.png)

**Nuxt version: 3.0.0-rc.12**

**Comment by victorgarciaesgi** [comment]
Oh that's strange! I will look at it but Nuxt keep pushing breaking changes it's hard to follow 😂
Thanks for reporting

**Comment by victorgarciaesgi** [comment]
Hi! How and where did you declare your `$sdk` ?

**Comment by victorgarciaesgi** [comment]
It should be fixed, I removed manual declaration and let Nuxt infer types from plugin.
I also added an option to disable plugin generation
